lib/beaker/options/parser.rb

```diff
@@ -181,9 +203,12 @@ def initialize
       # @raise [ArgumentError] Raises error on bad input
       def parse_args(args = ARGV)
         @options                        = @presets.presets
+        tag_sources(@presets.presets, "default")
         cmd_line_options                = @command_line_parser.parse(args)
         cmd_line_options[:command_line] = ([$0] + args).join(' ')
+        tag_sources(cmd_line_options, "flag")
         file_options                    = Beaker::Options::OptionsFileParser.parse_options_file(cmd_line_options[:options_file])
+        tag_sources(file_options, "file")
```

Comment:
we have two different file attribution values, so this should contain more detail. `options_file`, perhaps?